guides/common/modules/proc_upgrading-the-external-database-operating-system.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Upgrading_the_External_Database_Operating_System{context}"]
+= Upgrading the external database operating system
+
+If your {Project} uses an external database, you can upgrade the database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
```

Comment:
The external DB has more flexible requirements than Foreman since it just needs to be a running postgres server, so I think it's still okay that users would have EL8 machines running their DBs for some time. @ekohl can probably keep me honest here though.

Reply:
As @ianballou has indicated, it sort of has its own lifecycle. Technically I think we should loosen the requirement to any PostgreSQL 13+ compatible server instead of some (RH)EL server running PostgreSQL.

Within the platform team we've had some discussions on the level of support for the external DB server. Now that it's just a plain PostgreSQL server we can consider not having any instructions on setting up the PostgreSQL server. For example, we don't document how you do backups, partition, etc. Where do we draw the line?